tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -188,6 +194,44 @@ def test_update_snapshot(self):
                 **serialized_alert_rule,
             )
 
+    def _project_alert_rule_api_call(
```

Comment:
I've reworked when we uses `responses` and when we patch the Slack Client.
Whichever test uses this helper function _has_ to have configured the response or patched the client.
This function _only_ focuses on setting up the integration and making the API call.